app/views/home/tos_faq.html.erb

```diff
@@ -358,7 +358,7 @@ We understand that, as with many things, there are hard cases at the edges of ca
 
 
       <h5 class="heading" id="fanwork_commentary">What should I do about recs/commentary on fanworks?</h5>
-      <p><a href="https://archiveofourown.org/archive_faqs/15">Our bookmarking feature</a> is designed for your recommendations or commentary on specific works by other creators. You can bookmark fanworks hosted on the Archive or fanworks hosted elsewhere. Many creators also welcome discussion in the comments to the work, which is another appropriate place for such commentary. As always, while criticism of a fanwork is not itself harassment, content must comply with our other policies, including our harassment policy.
+      <p><%= link_to ts("Our bookmarking feature"), archive_faqs_path("15") %> is designed for your recommendations or commentary on specific works by other creators. You can bookmark fanworks hosted on the Archive or fanworks hosted elsewhere. Many creators also welcome discussion in the comments to the work, which is another appropriate place for such commentary. As always, while criticism of a fanwork is not itself harassment, content must comply with our other policies, including our harassment policy.
```

Comment:
This should be `archive_faq_path("15")`, since we're pointing to a specific FAQ and not the index.